### PR TITLE
Login added in react

### PR DESCRIPTION
**Pull request by Bhanu707**
please check this, although i am yet to apply best practice

### CODE REVIEW

File: ui/src/Dashboard/index.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    };
+
+    const handlePass = (e: any) => {
+        let val = e.target.value;
+        setState({ ...state, pass: val });
+    };
+
+    const redirectDashbord = () => {
+        props.LoginFun();
+        let url = "/home"
```
Comment (satyapriyamishra222) [inline ui/src/Dashboard/index.tsx:32]: Can be done in one line. No need of declaration

File: ui/src/Dashboard/index.tsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        history.push(url)
+    }
+    // useEffect(() => {
```
Comment (satyapriyamishra222) [inline ui/src/Dashboard/index.tsx:35]: Why commented code ?

File: ui/src/Dashboard/index.tsx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    //     if (localStorage.getItem('user-info')) {
+    //         history.push("/home");
+    //     }
+    // }, [])
+
+
+    const isVlaidLogin = () => {
+        if (state.email === 'admin' && state.pass === "admin") {
+            // console.log("valid user")
```
Comment (satyapriyamishra222) [inline ui/src/Dashboard/index.tsx:44]: remove commented code.

File: ui/src/Dashboard/index.tsx [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+            redirectDashbord()
+            return true
+        }
+        // console.log("Not valid user")
```
Comment (satyapriyamishra222) [inline ui/src/Dashboard/index.tsx:48]: Commented code

File: ui/src/Dashboard/index.tsx [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+        return false
+    }
+
+
+    const handleLogin = function (e: any) {
+        e.preventDefault();
+        isVlaidLogin()
+        //     loginSer.login(state).then((res) => {
```
Comment (satyapriyamishra222) [inline ui/src/Dashboard/index.tsx:56]: Commented code

File: ui/src/Dashboard/index.tsx [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+        //         console.log(error.message);
+        //     })
+        // }
+    }
+    return (<>
+        <Grid container className="h-100 login-wrp" direction="row" >
+            <Grid item xs={6} className="bg-img" >
+                <Grid container className="bg-mask" justifyContent="center" alignItems="center">
+                    <div className="promo-box">
+                        <h1 className="title"> Login Form </h1>
```
Comment (satyapriyamishra222) [inline ui/src/Dashboard/index.tsx:70]: Please bring these from i18n

File: ui/src/Dashboard/index.tsx [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+                        <p className="sub-title">Hi! This is EMS login page</p>
```
Comment (satyapriyamishra222) [inline ui/src/Dashboard/index.tsx:71]: No raw strings. Please bring from i18n

File: ui/src/Dashboard/index.tsx [hunk 8]
```diff
@@ -0,0 +1,94 @@
+import React, { useEffect } from 'react'
+import Grid from '@material-ui/core/Grid';
+import TextField from '@material-ui/core/TextField';
+// import loginSer from "./login.service";
```
Comment (satyapriyamishra222) [inline ui/src/Dashboard/index.tsx:4]: No commented code please. And there is no service concept in react.

File: ui/src/Dashboard/index.tsx (hunk 1 above)
Comment (Bhanu707) [reply ui/src/Dashboard/index.tsx:32]: converted to one line

File: ui/src/Dashboard/index.tsx (hunk 2 above)
Comment (Bhanu707) [reply ui/src/Dashboard/index.tsx:35]: removed commented code

File: ui/src/Dashboard/index.tsx (hunk 3 above)
Comment (Bhanu707) [reply ui/src/Dashboard/index.tsx:44]: removed

File: ui/src/Dashboard/index.tsx (hunk 4 above)
Comment (Bhanu707) [reply ui/src/Dashboard/index.tsx:48]: removed

File: ui/src/Dashboard/index.tsx (hunk 5 above)
Comment (Bhanu707) [reply ui/src/Dashboard/index.tsx:56]: removed

File: ui/src/Dashboard/index.tsx (hunk 6 above)
Comment (Bhanu707) [reply ui/src/Dashboard/index.tsx:70]: changed login and login form from i18n

File: ui/src/Dashboard/index.tsx (hunk 8 above)
Comment (Bhanu707) [reply ui/src/Dashboard/index.tsx:4]: removed imports

**Pull request merged**